packages/accounts-google/google.js

```diff
@@ -5,9 +5,19 @@ if (Meteor.isClient) {
     // support a callback without options
     if (! callback && typeof options === "function") {
       callback = options;
-      options = null;
+      options = {};
     }
 
+    // Use Google's domain-specific login page if we want to restrict creation to
+    // a particular email domain. (Don't use it if restrictCreationByEmailDomain
+    // is a function.) Note that all this does is change Google's UI ---
+    // accounts-base/accounts_server.js still checks server-side that the server
+    // has the proper email address after the OAuth conversation.
+    if (typeof Accounts._options.restrictCreationByEmailDomain === 'string') {
+      options.loginUrlParameters = _.extend(options.loginUrlParameters || {}, {
```

Comment:
This line looks like it may unexpectedly mutate `options.loginUrlParameters`.  Throw another `{}` at the beginning of the `_.extend` arguments?